Cargo.toml

```diff
@@ -34,3 +34,8 @@ proc-macro2 = "1.0.60"
 pkg-config = "0.3.27"
 cpal = "0.13"
 crossbeam = "0.8"
+
+[features]
+default = ["prerecorded", "live"]
+live = ["dep:tungstenite", "dep:tokio-tungstenite"]
```

Comment:
Use of `dep:` bumps our MSRV to 1.60. I don't think we document an MSRV, but we should and it'll need to be at least 1.60.

Reply:
Whoa.  rust 1.60.0 is from April 2022.

Reply:
I see the feature flags in a PR that never got merged.  Maybe that's what you were thinking of.

Reply:
That must be it